packages/hebao_v1/contracts/thirdparty/BytesUtil.sol

```diff
@@ -282,6 +283,18 @@ library BytesUtil {
         return tempBytes;
     }
 
+    // @dev This method was added by Daniel Wang.
+    function toBool(bytes memory _bytes, uint _start) internal pure returns (bool) {
```

Comment:
This seems to be used to decode the return value. This code likely works, but decoding the return value is ABI encoded so most of the time decoding it as harder. But there's a built in function that can be used for this:
```
abi.decode(data, (bool));
```

This is an easy way to decode whatever a function returns.